src/pages/steps/hooks/useRecentLocations.ts

```diff
@@ -38,6 +39,8 @@ const useRecentLocations = () => {
     .filter(
       (location) =>
         location &&
+        location?.workflowStatus !== WorkflowStatus.REJECTED &&
+        location?.workflowStatus !== WorkflowStatus.DELETED &&
         location?.name?.nl !== 'Online' &&
```

Comment:
Cant we drop the ? after location?

Reply:
No unfortunately some places don't have the attribute so it would return a false true and exclude them :/

Reply:
It was to skip a preliminary filter() loop but I can make the check a different way maybe